native/cocos/scene/ReflectionProbe.h

```diff
@@ -109,6 +117,9 @@ class ReflectionProbe final {
     inline void setPreviewPlane(Node* val) { _previewPlane = val; }
     inline const Node* getPreviewPlane() const { return _previewPlane; }
 
+    inline void setCubemap(cc::TextureCube* cubeMap) { _cubemap = cubeMap; }
+    inline const cc::TextureCube* getCubemap() { return _cubemap; }
```

Comment:
`inline const cc::TextureCube* getCubemap()` -> `inline const cc::TextureCube* getCubemap() const`

Reply:
And i think cube and map are two words. So It is better to rename to `getCubeMap`